Mail/Views/Thread/WebViewModel.swift

```diff
@@ -168,60 +168,80 @@ final class WebViewModel: NSObject, ObservableObject {
         }
     }
 
-    /// Adds breakpoints if the message contains words that are too long
-    /// Sometimes the WebView needs indication to break certain content like URLs
+    /// Adds breakpoints if the body contains text with words that are too long
+    /// Sometimes the WebView needs indication to break certain content like URLs, so the algorithm
+    /// inserts a `<wbr>` Element in places where a character string can be broken
     private func breakLongWords(of document: Document) throws {
-        guard let contentDiv = try document.getElementById(Constants.divWrapperId) else { return }
-        breakLongWords(of: contentDiv)
+        guard let contentDiv = document.body() else { return }
+        try breakLongWords(of: contentDiv)
     }
 
-    /// Walks through nodes and adds breakpoints to TextNode if necessary
-    private func breakLongWords(of node: Node) {
-        let childNodes = node.getChildNodes()
+    /// Walks through Element nodes and iterates over its TextNodes, then, if the text requires breakpoints
```

Comment:
I think it might just be "their TextNodes" instead of "its TextNodes" because there are multiple "Element nodes"